src/Access/Common/AuthenticationType.h

```diff
@@ -20,6 +20,9 @@ enum class AuthenticationType
     /// This kind of hash is used by the `mysql_native_password` authentication plugin.
     DOUBLE_SHA1_PASSWORD,
 
+    /// Password is encrypted in bcrypt hash.
+    BCRYPT_PASSWORD,
```

Comment:
It seems the authentication type is stored in the session log table as enum8 (i.e. as a byte), so probably it's better not to change underlying numeric constants of this enum.